backend/local/main.ts

```diff
@@ -0,0 +1,50 @@
+import * as WebSocket from 'ws';
+import { onConnect } from '../onconnect/src/on-connect';
+import { createTable, ddb } from './dynamo';
+import { onMessage } from '../sendmessage/src/on-message';
+
+interface WebsocketWithId extends WebSocket {
+  id?: string;
+}
+
+export function generateId(length: number): string {
+  const mask = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789';
+  let result = '';
+  for (let i = length; i > 0; --i) result += mask[Math.round(Math.random() * (mask.length - 1))];
+  return result;
+}
+
+createTable();
+
+const wss = new WebSocket.Server({
+  port: 8080,
+  host: 'localhost',
+});
```

Comment:
Would be nice to have a log that the server is running in port XXXX once it is running.

Reply:
On the other hand, maybe it is not needed as there is other log output already.